lib/settings.h

```diff
@@ -360,6 +360,9 @@ class CPPCHECKLIB Settings : public cppcheck::Platform {
     /** @brief XML version (--xml-version=..) */
     int xml_version;
 
+    /** @brief write SARIF results (--sarif) */
+    bool sarif;
```

Comment:
I think the members are more or less in alphabetical order so if you add it correspondingly..

Reply:
Done.